docs/pages/connect-your-client/vnet.mdx

```diff
@@ -32,6 +32,25 @@ features that traditional VPNs cannot provide.
 VNet is available on macOS and Windows in Teleport Connect and tsh, with plans
 for Linux support in a future version.
 
+<Admonition type="warning">
+VNet's VPN-like experience for app access means that any software running on
+the client machine can access Teleport apps at local DNS or IP addresses.
+
+**Avoid running VNet on shared or multi-user hosts.**
```

Comment:
```suggestion
**Avoid running VNet on shared or multi-user machines.**
```

People who don't know much about VNet typically don't understand immediately whether VNet is something that runs on their computers or on some servers, so I think using "machines" instead of "hosts" helps make it more clear in the context of this warning.